spark/v3.5/spark/src/main/java/org/apache/iceberg/spark/source/SparkPositionDeltaWrite.java

```diff
@@ -169,7 +174,18 @@ public DeltaWriterFactory createBatchWriterFactory(PhysicalWriteInfo info) {
       // broadcast the table metadata as the writer factory will be sent to executors
       Broadcast<Table> tableBroadcast =
           sparkContext.broadcast(SerializableTableWithSize.copyOf(table));
-      return new PositionDeltaWriteFactory(tableBroadcast, command, context, writeProperties);
+      return new PositionDeltaWriteFactory(
```

Comment:
Can we move the existing table broadcast and update the comment to be generic? So that both broadcasts are handled consistently. Something like below?

```
// broadcast large objects as the writer factory will be sent to executors
return new PositionDeltaWriteFactory(
    sparkContext.broadcast(SerializableTableWithSize.copyOf(table)),
    broadcastRewritableDeletes(),
    ...);
```

Reply:
Done!